src/mmw/apps/modeling/models.py

```diff
@@ -79,7 +79,7 @@ class Meta:
     name = models.CharField(
         max_length=255)
     project = models.ForeignKey(Project, related_name='scenarios')
-    current_condition = models.BooleanField(
+    is_current_conditions = models.BooleanField(
```

Comment:
I'm not sure I understand. The "current conditions" scenario is a special scenario that can't be modified. The model simulation to run depends on the project it is a part of. All scenarios in a project get run through the same model.